src/main/resources/beam-template.conf

```diff
@@ -391,7 +391,8 @@ beam.replanning.ModuleProbability_3 = 0.1
 beam.replanning.Module_4 = "TimeMutator"
 beam.replanning.ModuleProbability_4 = 0.0
 beam.replanning.fractionOfIterationsToDisableInnovation = "double | Double.PositiveInfinity"
-beam.replanning.cleanNonCarModesInIteration = "int | 0"
+beam.replanning.clearModesAtStartOfIteration.modes = [string] | []
+beam.replanning.clearModesAtStartOfIteration.atIteration = "int | 0"
```

Comment:
How should we turn it off? If we put 0, it will clean the modes at 0 iteration, correct? Shall we use optional or -1?

Reply:
Yeap, as it said in the task.
beam.replanning.clearModesAtStartOfIteration.atIteration="int | 0" (-1 is disabled)